includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -49,4 +79,43 @@ public static function track_store_setup() {
 
 		WC_Tracks::record_event( 'obw_store_setup', $properties );
 	}
+
+	/**
+	 * Track skipped steps.
+	 *
+	 * @return void
+	 */
+	public static function track_skip_step() {
+		$previous_step = get_option( 'woocommerce_obw_last_completed_step' );
+		$current_step  = isset( $_GET['step'] ) ? sanitize_text_field( $_GET['step'] ) : false; // phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification, WordPress.Security.ValidatedSanitizedInput
+		if ( ! $previous_step || ! $current_step ) {
+			return;
+		}
+
+		$wc_admin_setup_wizard_tracking = self::get_instance();
+		$steps                          = array_keys( $wc_admin_setup_wizard_tracking->steps );
+		$current_step_index             = array_search( $current_step, $steps, true );
+		$previous_step_index            = array_search( $previous_step, $steps, true );
+
+		// If we're going forward more than 1 completed step.
+		if ( $current_step_index > $previous_step_index + 1 ) {
+			$properties = array(
+				'step' => $steps[ $previous_step_index + 1 ],
+			);
```

Comment:
This code makes sense to me, but when I skip two steps in a row, the first skipped step is recorded for both events. Skipping `recommended` and `activate` yields `&step=recommended` for both events.

Reply:
Great catch, Paul.  Updated to track step name prior to current index in https://github.com/woocommerce/woocommerce/pull/22879/commits/e7c6e958456225d42b71df3612ab2e268132319c